operator/apis/loki/v1beta1/lokistack_types.go

```diff
@@ -766,6 +766,32 @@ type LokiStackComponentStatus struct {
 	Ruler PodStatusMap `json:"ruler,omitempty"`
 }
 
+// ObjectStorageStatusSchema defines the requirements needed to configure a new
+// storage schema status.
+type ObjectStorageStatusSchema struct {
```

Comment:
This is in the `v1beta1` API. Even when we decide to keep the new status struct in `v1`, I don't think we should change the old version of the API.